src/utils.ts

```diff
@@ -266,12 +266,13 @@ export const assetFromJSON = (asset: any): OpenSeaAsset => {
     openseaLink: asset.permalink,
     traits: asset.traits,
     numSales: asset.num_sales,
-    lastSale: {
+    lastSale: asset.last_sale ? {
```

Comment:
can we make a helper called `assetEventFromJSON` to do this object building, consistent with the other ones above?